### ISSUE TITLE
breakout when simplifying smoothed path

### ISSUE THREAD

**Issue by stinlemp** [paperjs/paper.js#620]
In some cases an edge of a smoothened and "handdrawn" path produces breakouts when simplifying it.

![bildschirmfoto 2015-01-14 um 12 35 08](https://cloud.githubusercontent.com/assets/7261560/5738210/de1d8d7e-9be9-11e4-8752-f59497ef576a.png)

settings of the path:
```javascript
path = new Path({
		segments: [event.point],
		strokeColor: 'black',
		strokeWidth: 10, 
		strokeCap: 'round',
		strokeJoin: 'round'
});
```

**Comment by lehni** [comment]
Sounds like the same as #414?

**Comment by stinlemp** [comment]
yes, seems like

**Comment by costa974** [comment]
how did you solve this? i have the same problem.

**Comment by lehni** [comment]
The fix from #414 is merged into the develop branch now:

https://github.com/paperjs/paper.js/commit/faecea3955fa23fcaaea5604b455f02da15f67b9
